Call to rsc->get_cur_state() incorrectly lists completed operations as failed (op_status=4)

LF Bug #1833
http://developerbugs.linux-foundation.org/show_bug.cgi?id=1833

Needs to go upstream

diff -r 98aeb5f25c95 lib/lrm/clientlib.c
--- a/lib/lrm/clientlib.c	Fri Feb 15 17:53:27 2008 +0100
+++ b/lib/lrm/clientlib.c	Fri Feb 15 17:53:56 2008 +0100
@@ -1190,10 +1190,6 @@ msg_to_op(struct ha_msg* msg)
 		else {
 			op->output = NULL;
 		}
-		/* check the rc to revise the op_status */
-		if (0 != op->rc) {
-			op->op_status = LRM_OP_ERROR;
-		}
 	} else if(op->op_status == LRM_OP_PENDING) {
 		op->rc = EXECRA_STATUS_UNKNOWN;
 		
